.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor
 node_modules
-.idea/
\ No newline at end of file
+.idea/
```

Comment:
@lykmapipo do you mind adding an `.editorconfig` file for this project to make coding style consistent

```
[*]
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = false
insert_final_newline = true
indent_style = space
indent_size = 4
```